pippo-core/src/main/java/ro/pippo/core/route/Route.java

```diff
@@ -155,7 +155,6 @@ public void setName(String name) {
 
     void setGroupUriPattern(String groupUriPattern) {
         if (absoluteUriPattern != null) {
-            // when group1.addRoute(route); group2.addRoute(route);
```

Comment:
It's just a simple comment. Maybe better is "when the route is added in both group1 and group2"